### ISSUE TITLE
Mobile Number Field verification warning appears immediately even though there was no action done

### ISSUE THREAD

**Issue by tutntut** [opengovsg/FormSG#4618]
<img width="652" alt="image" src="https://user-images.githubusercontent.com/103998698/185886460-448436fa-cf4d-417b-9274-76649e84d8dd.png">

**Comment by mantariksh** [comment]
can consider doing post-rollout if this is not easy to fix

**Comment by karrui** [comment]
ok investigated, not that easy to fix, and should this even be fixed? it is a real validation error on the field and is triggered on blur.

**Comment by justynoh** [comment]
this is hard to fix :( what we really want is to move the "Please verify this field" error to onblur of the box containing the input + button, but this removes onsubmit validation for the same thing.

**Comment by mantariksh** [comment]
yes we should definitely fix this. the user should not see an error if they have done nothing wrong. but we can do this after rollout
